src/Miki.Framework/ContextObject.cs

```diff
@@ -9,6 +12,11 @@
 	/// </summary>
     public interface IContext
 	{
+		/// <summary>
+		/// The message received from discord.
```

Comment:
"The message recieved from Discord" is deceiving, it will change into more providers in the future.